os_builders/roles/nubes_bootcontext/files/nubes-bootcontext.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Could we 
`set -euo pipefail` if we want to fail noisly?

---

Nope, failing noisily may mean that a user cant logon